invenio_app_rdm/records_ui/previewer/iiif_simple.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 data-futures.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Simple IIIF image preview."""
+
+from os.path import splitext
+
+from flask import current_app, render_template
+
+previewable_extensions = ['gif', 'jpg', 'jpeg', 'jp2', 'png', 'tif', 'tiff']
+
+# extensions that can be put in <img> tag - all others converted to jpg
+browser_extensions = ['gif', 'jpg', 'png']
```

Comment:
This could be a config (see similar comment for `supprorted_extensions` below)

Reply:
Done